docs/platforms/javascript/common/enriching-events/scopes/index.mdx

```diff
@@ -8,35 +8,97 @@ information from the current scope. SDKs will typically automatically manage the
 for you in the framework integrations and you don't need to think about them. However,
 you should know what a scope is and how you can use it for your advantage.
 
-## What's a Scope, What's a Hub
-
-You can think of the hub as the central point that our SDKs use to route an
-event to Sentry. When you call `init()` a hub is created and a client and a
-blank scope are created on it. That hub is then associated with the current
-thread and will internally hold a stack of scopes.
+## What's a Scope
 
 The scope will hold useful information that should be sent along with the
 event. For instance [contexts](../context/) or
 [breadcrumbs](../breadcrumbs/) are stored on
-the scope. When a scope is pushed, it inherits all data from the parent scope
-and when it pops all modifications are reverted.
+the scope. When a scope is forked, it inherits all data from the parent scope.
```

Comment:
l: I'm a bit on the fence if we should publicly speak about "forking" a scope. It's one of these terms that not all devs will understand immediately. Though to be honest, just saying "cloning" is probably not descriptive enough and something like "create scopes" doesn't imply that we inherit data... (especially in the next paragraph). Not sure, feel free to leave it as-is; this is an expert section anyway

Reply:
yeah, I know what you mean, I was also not 100% sure there... 🤔